embedded-database-spring-test/src/main/java/io/zonky/test/db/flyway/OptimizedFlywayTestExecutionListener.java

```diff
@@ -61,7 +64,7 @@
  *
  * @see <a href="https://www.postgresql.org/docs/9.6/static/manage-ag-templatedbs.html">Template Databases</a>
  */
-public class OptimizedFlywayTestExecutionListener extends FlywayTestExecutionListener implements Ordered {
+public class OptimizedFlywayTestExecutionListener extends org.flywaydb.test.FlywayTestExecutionListener implements Ordered {
```

Comment:
This change is not possible because it would not work with older versions. So, I've reverted this change.